python3/httplib2/certs.py

```diff
@@ -0,0 +1,32 @@
+"""
+certs
+
+Returns the path to the ca bundle from ca_certs_locater if available, or
+from certifi, if available, or the environmental variable HTTPLIB2_CA_CERTS,
+if available, or the default CA certificates file bundled with httplib2.
+
+Code originally from Requests library by Kenneth Reitz.
+
+"""
+__version__ = "1.0.0"
+__license__ = "MIT"
+__history__ = """
+"""
+
+import os
+import os.path
+
+try:
+    from certifi import where as get_cacerts
```

Comment:
`ca_certs_locater` is not mentioned here. Sorry, just found that it has same py2/3 difference in approved #52 

Should work same in py2/3.